third_party/silabs/efr32_sdk.gni

```diff
@@ -238,7 +266,8 @@ template("efr32_sdk") {
       "SL_RAIL_LIB_MULTIPROTOCOL_SUPPORT=1",
       "SL_RAIL_UTIL_PA_CONFIG_HEADER=<sl_rail_util_pa_config.h>",
       "RADIO_CONFIG_DMP_SUPPORT=1",
-
+      "SL_CATALOG_SLEEPTIMER_PRESENT",
+      "SL_CATALOG_POWER_MANAGER_PRESENT",
```

Comment:
Why are you adding it back here? They should be already defined in the board configuration and it was removed from here recently